src/components/Footer.tsx

```diff
@@ -1,11 +1,12 @@
-import { Box, Flex, Img, Link } from '@chakra-ui/react'
```

Comment:
You're removing spaces but the prettier action actually adds them. I suggest you run a `prettier --fix` at some point (and also to define it on save of your IDE...)